app/src/main/res/values/colors.xml

```diff
@@ -279,10 +279,12 @@
     <color name="toolbar_menu_transparent">@android:color/transparent</color>
 
     <!-- Mozilla Android Component color variable overrides -->
-    <color name="mozac_ui_icons_fill" tools:ignore="UnusedResources">?primaryText</color>
+    <color name="mozac_ui_icons_fill" tools:ignore="UnusedResources">@color/primary_text_light_theme</color>
```

Comment:
Will look into this. I don't think this will work for the private theme, which is why `?primaryText` was used.

Reply:
Yes you're right, I didn't think about the private mode. In that case the issue might be in AC where the icon is tinted.

Reply:
OK, I believe I found a (simple) fix in AC, I will make a PR there

Reply:
👀